### PR TITLE
1694 add jobs page and list new jobs

### LINKED ISSUE #1694: List new jobs opportunities
Add to https://kodadot.xyz/jobs so yeah could be pushed to `main-nuxt` 

We will be looking for 
- developer advocate -- very needed for a hackathon to start
--Description -> We need you to help us with incoming developers, creating tutorials for developers on how to hack on our stuff, making streamlined flow when bugs occur and the rest of the greatly known avocados. Making good demos. Being an english-native speaker is highly needed for a lot of educational videos
- Technical writer -- Creating technical writings to help better understand what we are doing with KodaDot and Metaprime

Add Discord link for **Learn more** https://discord.gg/35hzy2dXXh

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
jobs-page

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [x] I've tested it at </jobs>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1694
- [x] added `/jobs` to main-nuxt, copied over existing jobs and added 2 new jobs

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

File: pages/jobs.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+
+@Component({
+  name: 'JobsPage',
+  layout: 'full-width-layout'
+})
```
Comment (roiLeo) [inline pages/jobs.vue:30]: I didn't know you can put layout here, but `full-width-layout` was for landing page. 
Default layout might be good but you'll need to remove `<section class="section">` wrapper (see Faq)

**Review by roiLeo — COMMENTED** [review]
lgtm, I would suggest to add a link to jobs page in footer

File: pages/jobs.vue (hunk 1 above)
Comment (kkukelka) [reply pages/jobs.vue]: I see, just removed layout and added `/jobs` to `<Footer>` 👍 
Screenshot 2022-01-05 at 09 37 43


yes as far is i know you can actually put every Nuxt-Hook into the `@Component` part such as `head()` and `asyncData`

**Review by roiLeo — APPROVED** [review]
Top!

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
Hey, I mean we are still continuously looking for 
- Frontend Developers
- Technical Product Manager
- COO

Re https://kodadot.xyz/jobs
Can you add them back, please 🥺

I mean good hit that senior ops is rather COO, that was definitely dupe :)

**Comment by kkukelka** [comment]
> Hey, I mean we are still continuously looking for
> 
> * Frontend Developers
> * Technical Product Manager
> * COO
> 
> Re https://kodadot.xyz/jobs Can you add them back, please 🥺
> 
> I mean good hit that senior ops is rather COO, that was definitely dupe :)

Sure thing, just added them back

**Comment by yangwao** [comment]
> Hey, I mean we are still continuously looking for
> 
> * Frontend Developers
> * Technical Product Manager
> * COO
> 
> Re https://kodadot.xyz/jobs Can you add them back, please 🥺
> 
> I mean good hit that senior ops is rather COO, that was definitely dupe :)

just clarify, I haven't check links lol but now I know why you removed it! explainer in issue! :)

**Comment by yangwao** [comment]
Perfect, I’ve sent the payout https://kusama.subscan.io/extrinsic/0x592f6c5373039dde1016893323019fbbbb66ecf38a455cccedfc9f0d4d2d83d9

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**